### ISSUE TITLE
Lazy client getter in Processor

### ISSUE THREAD

**Issue by ngr** [sosw/sosw#276]
Implement something like this in `Processor`:

```
def get_client(self, name):
    if not getattr(self, name, None):
        setattr(self, name, self.register_client(name))
    
    return getattr(self, name)
```

**Comment by coderr01** [comment]
@ngr i like to contribute. Can you assign the issue

**Comment by ngr** [comment]
Sure. Are you human or AI?
Feel free to ask for assistance.

**Comment by ngr** [comment]
https://sosw.app/contribution/index.html

**Comment by coderr01** [comment]
@ngr  I am Human :)

**Comment by ngr** [comment]
@coderr01 are you still on this? :)

**Comment by ngr** [comment]
For DDB this is solved in https://github.com/sosw/sosw/pull/323
Available since ``0.7.45``
